packages/@ourworldindata/grapher/src/mapCharts/MapChart.tsx

```diff
@@ -1,5 +1,7 @@
 import React from "react"
+import { isNumber } from "lodash"
 import {
+    anyToString,
```

Comment:
```suggestion
import {
    isNumber,
    anyToString,
```

We need to be a bit careful here - we have the convention that we only import lodash via `@ourworldindata/utils`, because that's a bit better for tree shaking and slightly decreases the size of our assets.